src/boot/efi/meson.build

```diff
@@ -118,6 +118,7 @@ if have_gnu_efi
                         if value == '' or value == 'auto'
                                 cmd = 'if [ -e /etc/os-release ]; then . /etc/os-release; else . /usr/lib/os-release; fi; echo $@0@'.format(sbatvar[1])
                                 value = run_command(sh, '-c', cmd).stdout().strip()
+                                message('@0@ (from @1@): @2@'.format(sbatvar[0], sbatvar[1], value))
```

Comment:
maybe output as part of the status summary at the end of the build?

Reply:
That's make the whole more complex, because there's three pieces of information here… I don't think it's worth the trouble.

Reply:
well, i didn't mean that we should output how we got the data in the summary, but maybe show the whole set of sbat fields in the status output.